### ISSUE TITLE
Make a clear ```aria-label``` description for each icon action button in sidebar

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/calendar#4227]
### Is your feature request related to a problem? Please describe.

Accessibility issue.

Some action buttons in a left sidebar doesn't have ```aria-label``` attribute at all, some of them are very similar to others: it is not easy to recognize a difference via screen reader. Please check, that all icon action buttons in a left sidebar have this attribute and make a clear description for them.
